pyproject.toml

```
@@ -1,3 +1,3 @@
 [build-system]
-requires = ["setuptools", "Cython"]
+requires = ["setuptools", "Cython", "pkgconfig"]
```

This is not needed anymore, right?

```suggestion
requires = ["setuptools", "Cython"]
```

---

right, a leftover from an intermediate version